src/main.rs

```diff
@@ -86,8 +83,10 @@ fn main() {
     } else if args.len() == 3 && args[1] == "temp" {
         let temp = temp_userinput(args[2].clone());
         client.set_temp(temp).unwrap_or_else(|e| error(e));
-    } else {
+    } else if args.len() == 2 && args[1] == "help" {
         print_usage(&args[0]);
+    } else {
```

Comment:
I think we should explicitly check for the lack of args. If invalid args are passed, we should print the usage still.